store/docs/getting-started.md

```diff
@@ -0,0 +1,272 @@
+---
+title: "Getting Started"
+date: 2018-05-02T00:00:00+00:00
+weight: 20
+geekdocRepo: https://github.com/owncloud/ocis-store
+geekdocEditPath: edit/master/docs
+geekdocFilePath: getting-started.md
+---
+
+{{< toc >}}
+
+## Installation
+
+So far we are offering two different variants for the installation. You can choose between [Docker](https://www.docker.com/) or pre-built binaries which are stored on our download mirrors and GitHub releases. Maybe we will also provide system packages for the major distributions later if we see the need for it.
+
+### Docker
+
+TBD
+
+### Binaries
+
+TBD
+
+## Configuration
+
+We provide overall three different variants of configuration. The variant based on environment variables and commandline flags are split up into global values and command-specific values.
+
+### Envrionment variables
+
+If you prefer to configure the service with environment variables you can see the available variables below.
+
+#### Global
+
+STORE_CONFIG_FILE
+: Path to config file, empty default value
+
+STORE_LOG_LEVEL
+: Set logging level, defaults to `info`
+
+STORE_LOG_COLOR
+: Enable colored logging, defaults to `true`
+
+STORE_LOG_PRETTY
+: Enable pretty logging, defaults to `true`
+
+#### Server
+
+STORE_TRACING_ENABLED
+: Enable sending traces, defaults to `false`
+
+STORE_TRACING_TYPE
+: Tracing backend type, defaults to `jaeger`
+
+STORE_TRACING_ENDPOINT
+: Endpoint for the agent, empty default value
+
+STORE_TRACING_COLLECTOR
+: Endpoint for the collector, empty default value
+
+STORE_TRACING_SERVICE
+: Service name for tracing, defaults to `store`
+
+STORE_DEBUG_ADDR
+: Address to bind debug server
+
+STORE_DEBUG_TOKEN
+: Token to grant metrics access, empty default value
+
+STORE_DEBUG_PPROF
+: Enable pprof debugging, defaults to `false`
+
+STORE_DEBUG_ZPAGES
+: Enable zpages debugging, defaults to `false`
+
+STORE_HTTP_ADDR
+: Address to bind http server, defaults to `0.0.0.0:9461`
+
+STORE_HTTP_NAMESPACE
+: The http namespace
+
+STORE_HTTP_ROOT
+: Root path of http server, defaults to `/`
+
+#### Health
+
+STORE_DEBUG_ADDR
+: Address to debug endpoint
+
+### Commandline flags
+
+If you prefer to configure the service with commandline flags you can see the available variables below.
+
+#### Global
+
+--config-file
+: Path to config file, empty default value
+
+--log-level
+: Set logging level, defaults to `info`
+
+--log-color
+: Enable colored logging, defaults to `true`
+
+--log-pretty
+: Enable pretty logging, defaults to `true`
+
+#### Server
+
+--tracing-enabled
+: Enable sending traces, defaults to `false`
+
+--tracing-type
+: Tracing backend type, defaults to `jaeger`
+
+--tracing-endpoint
+: Endpoint for the agent, empty default value
+
+--tracing-collector
+: Endpoint for the collector, empty default value
+
+--tracing-service
+: Service name for tracing, defaults to `store`
+
+--debug-addr
+: Address to bind debug server
+
+--debug-token
+: Token to grant metrics access, empty default value
```

Comment:
Codacy found an issue: [[no-duplicate-headings] Do not use headings with similar content (32:1)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)